security/tutorials/app-layer-policy/enforce-policy-istio.md

```diff
@@ -159,33 +201,44 @@ If you still have your shell open in the customer pod, exit out or open a new te
 return to the customer pod later).
 
 ```bash
-kubectl apply -f \
+kubectl apply -n yaobank -f \
 {{ "/security/tutorials/app-layer-policy/manifests/20-attack-pod.yaml" | absolute_url }}
 ```
 
 Take a look at the [`20-attack-pod.yaml` manifest in your browser](manifests/20-attack-pod.yaml).
 It creates a pod and mounts `istio.summary` secret.  This will allow us to masquerade as if we were
 the `summary` service, even though this pod is not run as that service account.  Let's try this out.  First, `exec` into the pod.
 
-    kubectl exec -ti attack-<fill in pod ID> bash
+```bash
+# Determine attack pod name
+ATTACK_POD=$(kubectl get pod -l app=attack -n yaobank -o jsonpath='{.items[0].metadata.name}')
+
+# Execute bash command in attack container
+kubectl exec -ti $ATTACK_POD -n yaobank bash
+```
 
 Now, we will attack the database.  Instead of listing the contents like we did before, let's try
 something more malicious, like changing the account balance with a `PUT` command.
 
-    curl -k https://database:2379/v2/keys/accounts/519940/balance -d value="10000.00" \
-    -XPUT --key /etc/certs/key.pem --cert /etc/certs/cert-chain.pem
+```bash
+curl -k https://database:2379/v2/keys/accounts/519940/balance -d value="10000.00" \
+-XPUT --key /etc/certs/key.pem --cert /etc/certs/cert-chain.pem
+```
 
 Unlike when we did this with the customer web pod, we do not have Envoy to handle encryption, so we
 have to pass an `https` URL, the `--key` and `--cert` parameters to `curl` to do the cryptography.
 
 Return to your web browser and refresh to confirm the new balance.
 
-#### Network policy
+#### Policy
 
-We can mitigate both of the above deficiencies with a {{site.prodname}} policy.
+We can mitigate both of the above deficiencies with a {{site.prodname}} policy (if you still have your shell open in 
+the attack pod, first exit out or open a new terminal tab)
 
-    wget {{ "/security/tutorials/app-layer-policy/manifests/30-policy.yaml" | absolute_url }}
-    calicoctl create -f 30-policy.yaml
+```bash
+wget {{ "/security/tutorials/app-layer-policy/manifests/30-policy.yaml" | absolute_url }}
+kubectl create -n yaobank -f 30-policy.yaml
```

Comment:
This does not work. Resource in 30-policy.yaml are 
apiVersion: projectcalico.org/v3
kind: GlobalNetworkPolicy

If we want to use kubectl, those should be


apiVersion: crd.projectcalico.org/v1
kind: GlobalNetworkPolicy